charts/datahub/subcharts/datahub-frontend/templates/deployment.yaml

```diff
@@ -49,7 +49,8 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          lifecycle: {{ .Values.lifecycle }}
+          lifecycle:
+            {{ - toYaml .Values.lifecycle | nindent 12 }}
```

Comment:
helm linter fails here. 

could you explain what are you trying to achieve with this? or what's the problem you are fixing?